src/ol_infrastructure/applications/xpro/Pulumi.applications.xpro.QA.yaml

```diff
@@ -23,9 +23,11 @@ config:
     MITOL_HUBSPOT_API_ID_PREFIX: "xpro-rc"
     MITXPRO_BASE_URL: "https://rc.xpro.mit.edu"
     MITXPRO_LOG_LEVEL: "INFO"
+    OPENEDX_OAUTH_PROVIDER: "ol-oauth2"
```

Comment:
`MITXPRO_OAUTH_PROVIDER` will be renamed to `OPENEDX_OAUTH_PROVIDER` after the merge of https://github.com/mitodl/mitxpro/pull/3327